### ISSUE TITLE
WCAG Issue - Theming - Click target of checkbox overlapping thumbnail is too small

### ISSUE THREAD

**Issue by Rainribbon** [Kaltura-EDU/issues-tracking#39]
Submitted as Kaltura Support Case #00592509:

Today I encountered another accessibility issue with the checkboxes overlapping the media thumbnails in My Media: it's too easy to click on the wrong element (thumbnail vs. checkbox). The size of the checkbox is 16 x 16 pixels. This is a failure of [SC 2.5.8 Target Size (Minimum) (Level AA)](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html)

<img width="830" height="283" alt="Screenshot of the interface annotated with an arrow to show how inadvertently clicking a pixel or two to the right of the checkbox makes you click on the thumbnail." src="https://github.com/user-attachments/assets/a5451be2-3234-4c50-8090-123fee4b7082" />

**Comment by Rainribbon** [comment]
Related to Issue #20
